lib/schemas.js

```diff
@@ -1,11 +1,12 @@
-const joi = require('joi');
+const joi = require('@hapi/joi');
 
 const FIELD = joi.string().min(1);
 
 const ELASTICSEARCH_SCHEMA = joi.object({
   bulk: joi.object({
     body: joi.any().forbidden()
-  }).optional().unknown()
+  }).optional().unknown(),
+  apiVersion: joi.string().default('5')
```

Comment:
There is no need for this config any more

Reply:
Ah yes, missed that one. Thanks.